src/ncp/spinel.c

```diff
@@ -393,9 +393,9 @@ spinel_datatype_vunpack_(const uint8_t *data_ptr, spinel_size_t data_len, const
                 *arg_ptr = (const char *)data_ptr;
             }
 
-            ret += (spinel_size_t)len;
```

Comment:
Is there a reason you removed these? I believe I had to add them to fix warnings on Windows when building NCP.